internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -233,6 +252,17 @@ func isEnabled(request *common.Request) bool {
 	return isEnabled
 }
 
+func getVmConsoleProxyNamespace(request *common.Request) string {
+	if request.Instance.GetAnnotations() == nil {
+		return "kubevirt"
+	}
+	namespaceStr, ok := request.Instance.GetAnnotations()[VmConsoleProxyNamespaceAnnotation]
+	if !ok {
+		return "kubevirt"
```

Comment:
Can you extract the `"kubevirt"` string literal to a constant? Then we can be sure that both branches return the same string, and there is not a typo.

Reply:
Yep, done.